### ISSUE TITLE
Assertion failure in StandardReports

### ISSUE THREAD

**Issue by macumber** [NREL/OpenStudio#1952]
[BOOST_ASSERT] <2> Assertion ok failed on line 108 of class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl openstudio::toNeatString(double,unsigned int,bool) in file E:\openstudio\openstudiocore\src\utilities\core\StringHelpers.cpp.

**Comment by macumber** [comment]
I hit this when running an empty model after setting only the weather file

**Comment by macumber** [comment]
-1.#IND000000000000 is being passed to toNeatString

**Comment by DavidGoldwasser** [comment]
@macumber thanks, I'll look into this, but to be honest I'm surprised an empty model doesn't fail in EnergyPlus, although I guess you can run a model with no thermal zones, just can't run with with a thermal zone that doesn't have surfaces.

**Comment by macumber** [comment]
The main use case for running an empty model is to get E+ to calculate construction heat transfer properties.  I also do it because it is fast for testing :-)

**Comment by DavidGoldwasser** [comment]
I"ll add an empty model to unit test for measure.

**Comment by DavidGoldwasser** [comment]
@macumber for reference, the measure still runs fine, producing an HTML file despite the boost error being logged. But I'll still try to see what is being passed to StringHelpers.cpp that causes the error to be thrown.

**Comment by DavidGoldwasser** [comment]
@macumber I fixed this in the measure by not calculating EUI if the building doesn't have any area, it seems to be that 'OpenStudio.toNeatString(value, 2, true)' should be able to handle NaN as a value, maybe by returning an empty string? At any rate, the measure now bypasses this.

@henryHorsey related to this. I have a runner.registerValue for EUI, which now gets skipped from a datapoint if the building doesn't have any floor area. Since there may be exterior lights or equipment the EUI isn't necessarily 0. Is it ok to skip the registerValue or do I need to pass it on with some non double value. It didn't let me pass in nil as a value, so I just deleted it for now.
